components/clientComponents/globals/Captcha/helpers.ts

```diff
@@ -0,0 +1,14 @@
+"use client";
+
+export const hCaptchaEnabled = (featureFlag: boolean, isPreview: boolean = false) => {
+  if (
+    // process.env.NODE_ENV === "development" ||
```

Comment:
Remember to put un-comment this before merging to staging. For now, this allows testing locally but after this test case there is not really any reason to test locally.